doc/source/io.rst

```diff
@@ -1962,6 +1962,41 @@ advanced strategies
 Reading Excel Files
 ~~~~~~~~~~~~~~~~~~~
 
+.. versionadded:: 0.17
+
+``read_excel`` can read a ``MultiIndex`` index, by passing a list of columns to ``index_col``
+and a ``MultiIndex`` column by passing a list of rows to ``header``.  If either the ``index``
+or ``columns`` have serialized level names those will be read in as well by specifying
+the rows/columns that make up the levels.
+
+.. ipython:: python
+
+    # MultiIndex index - no names
+    df = pd.DataFrame({'a':[1,2,3,4], 'b':[5,6,7,8]},
+                      index=pd.MultiIndex.from_product([['a','b'],['c','d']]))
+    df.to_excel('path_to_file.xlsx')
+    df = pd.read_excel('path_to_file.xlsx', index_col=[0,1])
+    df
+
```

Comment:
I like adding your pictures of the before after here as well (the onces from above)